frontend/src/components/modals/category/ModalCreateCategory.jsx

```diff
@@ -0,0 +1,130 @@
+import { Modal, Button, Form, Card } from 'react-bootstrap';
+import { useTranslation } from 'react-i18next';
+import { useState, useEffect } from 'react';
+import axios from 'axios';
+import { useAuth } from '../../context/AuthContext';
+
+const ModalCreateCategory = ({ showModal, onClose, uuid }) => {
+  const [t] = useTranslation();
+  const { getTokenHeader } = useAuth();
+  const [photoLimitError, setPhotoLimitError] = useState('');
+  const [formData, setFormData] = useState({
+    category_name_lt: '',
+    category_name_en: '',
+    category_description_lt: '',
+    category_description_en: '',
+    album_type: '',
+    photo_limit: ''
+  });
+
+  const createCategory = async () => {
+    try {
+      await axios.post(`http://localhost:8080/api/v1/category/competition/${uuid}`, formData, {
+        headers: getTokenHeader()
+      });
+      setFormData({
+        category_name_lt: '',
+        category_name_en: '',
+        category_description_lt: '',
+        category_description_en: '',
+        album_type: '',
+        photo_limit: ''
+      });
+    } catch (error) {
+      alert(t('modalCreate.error'), error);
+    }
+    onClose();
+  };
+
+  const handleInputChange2 = (event) => {
+    const { name, value } = event.target;
+    if (name === 'photo_limit') {
+      if (value === '' || (parseInt(value) >= 1 && parseInt(value) <= 15)) {
+        setFormData({
+          ...formData,
+          [name]: value
+        });
+        setPhotoLimitError('');
+      } else {
+        setPhotoLimitError(t('editcomp.limitError'));
+      }
+    } else {
+      setFormData({
+        ...formData,
+        [name]: value
+      });
+    }
+  };
+  return (
+    <Modal show={showModal} onHide={onClose}>
+      <Modal.Header closeButton>
+        <Modal.Title>{t('modalCreate.titleCreate')}</Modal.Title>
+      </Modal.Header>
+      <Modal.Body>
+        <Form.Label> {t('modaleEdit.titleCat_en')}</Form.Label>
```

Comment:
sukeisti taip, kad pirma eitu LT pavadinimas po to EN:
![image](https://github.com/Vitalij-C-Techin/LietuvosSpaudosFotografija/assets/150675909/2b04dfbe-9ed3-463f-bd7e-ee3f3bf3f698)